core/src/validator.rs

```diff
@@ -346,6 +350,9 @@ impl ValidatorConfig {
             enforce_ulimit_nofile: false,
             rpc_config: JsonRpcConfig::default_for_test(),
             block_production_method: BlockProductionMethod::ThreadLocalMultiIterator,
+            replay_forks_threads: NonZeroUsize::new(1).expect("1 is non-zero"),
+            replay_transactions_threads: NonZeroUsize::new(get_max_thread_count())
+                .expect("thread count is non-zero"),
```

Comment:
how confident are we in this expect? i know we _physically_ have at least one by virtue of the code running, but that doesn't make the `num_cpus` (or w/e is backing this method call) platform implementation bug-free